config/database.appveyor.yml

```diff
@@ -0,0 +1,32 @@
+# SQLite version 3.x
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem 'sqlite3'
+#
+default: &default
+  adapter: postgresql
+  encoding: unicode
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  username: postgres
+  password: Password12!
+  host: localhost
+
+
+development:
```

Comment:
I guess the `development` and `production`  sections will not be relevant for this database config.